app-modules/ai/src/Filament/Pages/ManageAiIntegrationsSettings.php

```diff
@@ -97,6 +99,10 @@ public function form(Form $form): Form
                                     ->autocomplete(false),
                                 TextInput::make('open_ai_gpt_35_model')
                                     ->label('Model'),
+                                Select::make('open_ai_gpt_35_applicable_features')
```

Comment:
For each of these new select inputs, would it be good to apply `enum` to them? Does that not then add a validation rule for the ENUM?

Reply:
Yeah done